src/averageAges.js

```diff
@@ -1,61 +1,59 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+function Men(people) {
```

Comment:
```suggestion
function men(people) {
```

Reply:
`Men` is for classes, here you've got just a function, that helps you to filter people by sex

Reply:
moreover, I'd do something like this
```suggestion
function getBySex(people, sex) { // people – array, sex – 'm' | 'f'
```
agree, if you had used this approach, you would have avoided creating two functions – `Men`/`Women`